libs/vre/advanced-search/src/lib/feature/advanced-search/advanced-search.component.ts

```diff
@@ -233,4 +239,32 @@ export class AdvancedSearchComponent implements OnInit {
     handleChildValueChanged(property: ParentChildPropertyPair): void {
         this.store.updateChildSearchValue(property);
     }
+
+    loadPreviousSearch(): void {
+        if(this.previousSearchObject) {
```

Comment:
remove log,
less indent: if(!this.previousSearchObject) {
           return;
}

Reply:
done in [e8c04f9](https://github.com/dasch-swiss/dsp-das/pull/1226/commits/e8c04f996bfd74578787aab69d38c9b799f67c26)